stacks/camunda-ci-oracle-23/docker-stack.yml

```diff
@@ -0,0 +1,14 @@
+---
+services:
+  main:
+    command: /usr/local/bin/start-container.sh
+    deploy:
+      restart_policy:
+        condition: on-failure
+    environment: []
+    image: europe-west1-docker.pkg.dev/ci-30-162810/main/oracle:23v0.1.0
+    ports:
+      - 1521/tcp
+      - 22/tcp
```

Comment:
If you mean the 22 port, all other oracle templates have it defined even though the port is not exposed from the base image so I thought it might be necessary for regression. I removed it now though :)

Reply:
This is for instance the inspection of the 19v.0.2.2 image:

> ➜  ~ (int:sales) docker inspect europe-west1-docker.pkg.dev/ci-30-162810/main/oracle:19v0.2.2
> [
>     {
>         "Id": "sha256:565c8b8d19e1bc0a6bc8ed7d9b8f7e34328af08cedf53ebbba5454f84cbb8bed",
>         "RepoTags": [
>             "europe-west1-docker.pkg.dev/ci-30-162810/main/oracle:19v0.2.2"
>         ],
>         "RepoDigests": [
>             "europe-west1-docker.pkg.dev/ci-30-162810/main/oracle@sha256:13eb9697d8f04b8f7c330b313b36f932c0495ffd723d9f3ed8452bedfd3d1ce3"
>         ],
>         "Parent": "",
>         "Comment": "",
>         "Created": "2020-01-09T07:44:04.11530608Z",
>         "DockerVersion": "18.06.0-ce",
>         "Author": "",
>         "Config": {
>             "Hostname": "",
>             "Domainname": "",
>             "User": "",
>             "AttachStdin": false,
>             "AttachStdout": false,
>             "AttachStderr": false,
>             "ExposedPorts": {
>                 "1521/tcp": {},
>                 "5500/tcp": {},
>                 "8080/tcp": {}
>             },
>             "Tty": false,
>             "OpenStdin": false,
>             "StdinOnce": false,
>             "Env": [
>                 "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
>                 "LANG=en_US.UTF-8",
>                 "LANGUAGE=en_US.UTF-8",
>                 "LC_ALL=en_US.UTF-8",
>                 "GCLOUD_SDK_BIN=/opt/google-cloud-sdk/bin",
>                 "JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk",
>                 "M2_HOME=/opt/maven",
>                 "NODE_VERSION=0.10.38",
>                 "NPM_VERSION=2.10.1",
>                 "NVM_VERSION=0.33.2",
>                 "NVM_DIR=/usr/local/nvm",
>                 "WORK_DIR=/home/jenkins",
>                 "NODE_PATH=/usr/local/nvm/v0.10.38/lib/node_modules",
>                 "ORACLE_BASE=/opt/oracle",
>                 "ORACLE_HOME=/opt/oracle/product/19c/dbhome_1",
>                 "ORACLE_SID=ORCLCDB",
>                 "ORACLE_DOCKER_INSTALL=true",
>                 "DB_USER=camunda",
>                 "DB_PASSWORD=camunda"
>             ],
>             "Cmd": null,
>             "ArgsEscaped": true,
>             "Image": "sha256:fa30ec8f90b5d923015a989e426c5d1d1ead5db078d6e14d87a3d327338c7149",
>             "Volumes": {
>                 "/home/jenkins/.jenkins": {}
>             },
>             "WorkingDir": "/home/jenkins",
>             "Entrypoint": [
>                 "/usr/local/bin/start-container.sh"
>             ],
>             "OnBuild": null,
>             "Labels": {
>                 "org.label-schema.schema-version": "= 1.0     org.label-schema.name=CentOS Base Image     org.label-schema.vendor=CentOS     org.label-schema.license=GPLv2     org.label-schema.build-date=20180531",
>                 "repo": "https://github.com/camunda-ci/camunda-docker-ci-oracle"
>             },
>             "Shell": [
>                 "/bin/bash",
>                 "-euxo",
>                 "pipefail",
>                 "-c"
>             ]
>         },
>         "Architecture": "amd64",
>         "Os": "linux",
>         "Size": 15933634076,
>         "GraphDriver": {
>             "Data": {
>                 "LowerDir": "/var/lib/docker/overlay2/5c23b59a94e6ebfbaf60155bdb1472b85a27f335f2a5b6efea247059f1cd2f37/diff:/var/lib/docker/overlay2/ef99e0a6e1a7f08e1953f12d53a679e2e72307ecd17c36e46291f81e5517cb9f/diff:/var/lib/docker/overlay2/9e0f9cfbd70a2178b92104780ded0296c25e9bf46eae6a53889e89f6c1927ad3/diff:/var/lib/docker/overlay2/a13211851ce2cc81fd8a4e1087fdebe703bf249696576aebda119428f1a76033/diff:/var/lib/docker/overlay2/71f483372a31eff90e00b0b3808489b746ae1246d0de72cc336a964a972cb228/diff:/var/lib/docker/overlay2/7d30f09dd7c09bbd24dc979f70c8ce5fda1689931ae429d48044b3b9cc001d2b/diff:/var/lib/docker/overlay2/1c69ddc3e48870b078152fd00ac58f6934c610cefa5b3ba28b891dd7921ac315/diff:/var/lib/docker/overlay2/b89bd27e9e5959c190d837b2085bd65179b82a9f46f4796a4c35641e194ec46d/diff:/var/lib/docker/overlay2/f7b6e5a7307cde065b9ae0d7501635f10d230284e8c1862a6e01bc710b7c537c/diff:/var/lib/docker/overlay2/8d244ae69ba78bf69ebff3104527962168e69b33cdbe45608904d597fe8a1691/diff:/var/lib/docker/overlay2/c7022517de92a73bfe0a00bb0ef07f72a727a24cb9362d29947009c5b46fc738/diff:/var/lib/docker/overlay2/4e79f7dd4d53d8336f3a829209e79eee5dc21cad2baca737c3cf19d8ee2e0819/diff:/var/lib/docker/overlay2/d7d8bb18eee7b88f75b670581e50340bd48c7a3476912d77d73a7f97d8ee0632/diff:/var/lib/docker/overlay2/058af0ea9e6abfef4f7225cc42e57bddad81e7b677b80984bebbb0073df47446/diff:/var/lib/docker/overlay2/1423bdab10842817555e24bce0595193fbe52c2c111ee0233092a4186bcf9579/diff:/var/lib/docker/overlay2/da72978e21e0208ee40d7a150f8c24d2238c32b3d61f73f1595bc7e21acc5cc2/diff:/var/lib/docker/overlay2/d0d559c19fac42fe9411fef9a781e84da7633c1133fd91605b97062b13cb9605/diff:/var/lib/docker/overlay2/a1eead821a373de6a3a12a9b0a19c54d2f161ae72873c803742a1ec2b2bccf38/diff:/var/lib/docker/overlay2/294355ce40af34cf9507aad3fa8cc925c2477e10bc01d64a35d5a6f45e77ecfe/diff",
>                 "MergedDir": "/var/lib/docker/overlay2/2eae2be3105f4fe25832d033b579ded61eb74b19a5777065156a3dd2d966de0b/merged",
>                 "UpperDir": "/var/lib/docker/overlay2/2eae2be3105f4fe25832d033b579ded61eb74b19a5777065156a3dd2d966de0b/diff",
>                 "WorkDir": "/var/lib/docker/overlay2/2eae2be3105f4fe25832d033b579ded61eb74b19a5777065156a3dd2d966de0b/work"
>             },
>             "Name": "overlay2"
>         },
>         "RootFS": {
>             "Type": "layers",
>             "Layers": [
>                 "sha256:4826cdadf1ef360641a2f2745453e2e8bb30e3e82a683f4d1145e94bf1c18d9f",
>                 "sha256:7593b4a24886e8fe2f7336fa2d5ffe2c9737b11a0f70f818cdf12624a7e5013c",
>                 "sha256:7b1733c9f61bb05de0f8f00bf37aa38a44e017c8021bf86231c294bf80132407",
>                 "sha256:c19276f9933d6968b6c01ba9c5328c425dc7b1112b8fc342804a1496c3c333a2",
>                 "sha256:4a6631e99fe53668e4e05d62885f119bd1d8d20aeddfcd9508027c41d60559fc",
>                 "sha256:8a134bc7e77cc9fb19a13ad9872579a181bd6fc7b3ebfbc61f123fe4dc5185fb",
>                 "sha256:ee548d15f04398c7f6a5a225112a1ddad066d1ac5cdcaf7be951a631d0f0a613",
>                 "sha256:4c0107c4ff7f220b963dd787277f9648e5433b48bf9156edba0339303f7d28ce",
>                 "sha256:5ab13716178194642c35c057cd9dc83eddd76cbe6976919b73ee493fbe2ef0a8",
>                 "sha256:c91a60343f7c51c6c41810bc81fb6a33a60fd8c293de02ebe81aaa46eb325ce6",
>                 "sha256:b0d84e9f3a24cd1a3050f2eeb9f7f64ba2a2533781fbf05ce2b7092c12a91727",
>                 "sha256:8a17dec507b0f53dfecd399aa6be9fdfcb8dc210e4ffe06e960a26db978e787e",
>                 "sha256:3be7e1d2376791714205d656f144ca79f07afbe9d4c7360a5cbc9ed1ff85f977",
>                 "sha256:203869f6d2250d42240f071d1577ea30fd1cd5651096a03622f74a5aa48670c2",
>                 "sha256:83cb29cde3cc35ba688f6c240476aa910b906efa5e228b81ee4031740c8d11f5",
>                 "sha256:9aeb3d88bf10417b2233e3c2e90751264ed6f777c0b69d15fb4955da0b80a4e6",
>                 "sha256:d958a574cd60c6204e22505b4cdd3b4efec393066d3bd8a5ed79da34ab483605",
>                 "sha256:ea76bf7cd9ac5ab486040964fc009999934e1fc3876b488834440ba14f3a9a5c",
>                 "sha256:d50bb0a364943876033a402b10e9d8997bcb7f31f282a26a467e446496df5b59",
>                 "sha256:faaa5bffb240f0cf167508063c3c7464462b515f551f7509c311d38a77138f9c"
>             ]
>         },
>         "Metadata": {
>             "LastTagTime": "0001-01-01T00:00:00Z"
>         }
>     }
> ]